README.md

```diff
@@ -9,3 +10,13 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Two graphs are isomorphic if they have a one-to-one and onto relationship. If A and B are completely connected, then each vertice in both A and B has a unique edge that connects to every other vertice in the
+graph. If both A and B are completely connected, and have the same number of nodes, then A and B have the same number of edges. If both graphs are compeletely connected, then any node in A can map by a function
```

Comment:
What does this mapping look like? It sounds like you're just assuming that it exists.